package.json

```diff
@@ -14,7 +14,8 @@
 		"@supabase/supabase-js": "^1.35.6",
 		"next": "12.2.5",
 		"react": "18.2.0",
-		"react-dom": "18.2.0"
+		"react-dom": "18.2.0",
+		"react-icons": "^4.4.0"
```

Comment:
@ykdojo @iShibi @subhoghoshX 
are you guys fine with `react-icons` package? IMO i could have downloaded svgs(fontawesome) and place it inside public folder and use it instead of adding a package.

Reply:
I don't have a strong opinion on it as long as the performance is reasonably fast!

Reply:
Their docs state that tree-shaking works with this library. If that's true then it is fine to keep it as a dependency.